src/main/java/bio/terra/cli/command/resource/create/GcpNotebook.java

```diff
@@ -228,18 +227,14 @@ protected void execute() {
             .populateMetadataFields()
             .stewardshipType(StewardshipType.CONTROLLED)
             .accessScope(AccessScope.PRIVATE_ACCESS);
-    Map<String, String> allMetadata = defaultMetadata(Context.requireWorkspace().getId());
-    if (metadata != null) {
-      allMetadata.putAll(metadata);
-    }
     CreateGcpNotebookParams.Builder createParams =
         new CreateGcpNotebookParams.Builder()
             .resourceFields(createResourceParams.build())
             .instanceId(instanceId)
             .location(location)
             .machineType(machineType)
             .postStartupScript(postStartupScript)
-            .metadata(allMetadata);
+            .metadata(Optional.ofNullable(metadata).orElse(Collections.EMPTY_MAP));
```

Comment:
nit: `EMPTY_MAP` is a raw type. I think this should be `Collections.emptyMap()`, which is just a type-safe way to get the same map.